src/navigation/navigation_main.cc

```diff
@@ -690,6 +704,42 @@ void LoadConfig(navigation::NavigationParameters* params) {
   params->target_vel_tolerance = CONFIG_target_vel_tolerance;
   params->target_angle_tolerance = CONFIG_target_angle_tolerance;
   params->local_fov = CONFIG_local_fov;
+  params->use_kinect = CONFIG_use_kinect;
+  params->model_path = CONFIG_model_path;
+  params->evaluator_type = CONFIG_evaluator_type;
+
+  if (params->use_kinect) {
+    params->K = { 622.90532,   0.     , 639.44796, 0.     , 620.84752, 368.20234, 0.     ,   0.     ,   1.     };
+    params->D = { 0.092890, -0.046208, 0.000622, -0.001104, 0.000000 };
```

Comment:
ideally these would come from config directly, but encoding matrices in the `lua` file was annoying. Does anyone have a better solution?

Reply:
@srabiee had a good way to load camera configs - can you please suggest the change?

Reply:
@kvsikand it is common to have a separate yaml file for camera calibration. The file would be generated directly by camera calibration tools. If you do use the yaml format, then you can use either of these two helper functions depending on whether you want to use the camera matrix as a CV mat or an Eigen mat:
https://github.com/ut-amrl/cpip/blob/main/shared/src/depth2pointcloud.cpp#L547
https://github.com/ut-amrl/cpip/blob/main/shared/src/depth2pointcloud.cpp#L622